policy.go

```diff
@@ -615,13 +803,6 @@ func computePolicyORData(alg tpm2.HashAlgorithmId, trial *tpm2.TrialAuthPolicy,
 // computeDynamicPolicy computes the part of an authorization policy associated with a sealed key object that can change and be
 // updated.
```

Comment:
could also have a short description that is the OR of the pcr values plus a check on the optional counter being <=